### PR TITLE
data:shivaabhishek07

### PR DESCRIPTION
**Pull request by shivaabhishek07**
## Changes proposed
- I have created my own linkfree profile

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2860"></a>

### CODE REVIEW

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
missing commas after every link object

File: data/shivaabhishek07.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Intrested in DevOps | Open Source",
+  "avatar": "https://github.com/shivaabhishek07.png",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/shivaabhishek71",
+      "icon": "FaTwitter"
+    }
```
Comment (Aadarsh805) [inline data/shivaabhishek07.json:12]:
```
{
  "name": "Shiva Abhishek",
  "displayStatsPublic": true,
  "type": "personal",
  "bio": "Intrested in DevOps | Open Source",
  "avatar": "https://github.com/shivaabhishek07.png",
  "links": [
    {
      "name": "Twitter",
      "url": "https://twitter.com/shivaabhishek71",
      "icon": "FaTwitter"
    },
    {
      "name": "Github",
      "url": "https://github.com/shivaabhishek07",
      "icon": "FaGithub"
    },
    {
      "name": "LinkedIn",
      "url": "www.linkedin.com/in/shivaabhishek71",
      "icon": "FaLinkedin"
    },
    {
      "name": "Blogs",
      "url": "https://shiva71.hashnode.dev/",
      "icon": "SiHashnode"
    }
  ]
}
```
Should be like this

**Comment by Aadarsh805** [comment]
@shivaabhishek07 and your LinkedIn profile link is wrong, it should be `https://www.linkedin.com/in/shivaabhishek71/`

File: data/shivaabhishek07.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    {
+      "name": "Github",
+      "url": "https://github.com/shivaabhishek07",
+      "icon": "FaGithub"
+    }
+    {
+      "name": "LinkedIn",
+      "url": "www.linkedin.com/in/shivaabhishek71",
```
Comment (Aadarsh805) [inline data/shivaabhishek07.json:20]: LinkedIn URL should be `https://www.linkedin.com/in/shivaabhishek71`

**Review by Aadarsh805 — APPROVED** [review]
All good  👍

File: data/shivaabhishek07.json [hunk 3]
```diff
@@ -25,5 +25,10 @@
       "url": "https://shiva71.hashnode.dev/",
       "icon": "SiHashnode"
     }
```
Comment (Aadarsh805) [inline data/shivaabhishek07.json:27]: another comma missing here

File: data/shivaabhishek07.json [hunk 4]
```diff
@@ -25,5 +25,10 @@
...
+    {
+      "name": "Checkout my DevOps learnings repo",
+      "url": "https://www.github.com/shivaabhishek07/DevOps",
+      "icon": "FaGithub"
+    },
```
Comment (Aadarsh805) [inline data/shivaabhishek07.json:32]: and this comma must be removed

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

**Review by Aadarsh805 — APPROVED** [review]
All good here 👍

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**